src/lib/errors/UnprocessedItems.ts

```diff
@@ -0,0 +1,24 @@
+export class UnprocessedItemsException {
+	constructor(message: string) {
+		const error = Error(message);
+
+		Object.defineProperty(error, 'message', {
+			get() {
+				return message;
+			}
+		});
+		Object.defineProperty(error, 'name', {
+			get() {
+				return 'UnprocessedItemsException';
+			}
+		});
+		Object.defineProperty(error, 'code', {
+			get() {
+				return 'UnprocessedItemsException';
+			}
+		});
+
+		Error.captureStackTrace(error, UnprocessedItemsException);
+		return error;
+	}
+}
```

Comment:
This is a bit different than we do it in the monorepo.

```ts
export class UnprocessedItemsException extends Error {
	readonly name: 'UnprocessedItemsException' = 'UnprocessedItemsException';
	readonly code: 'UnprocessedItemsException' = 'UnprocessedItemsException';

	constructor(message: string) {
		super(message);

		// Restore prototype chain
		Object.setPrototypeOf(this, new.target.prototype);
	}
}
```

this should allow you to say that `error instanceof UnprocessedItemsException` which is not possible with the current implementation.

Reply:
Also, rename the file to `unprocessed-items-exception.ts` ;).